### PR TITLE
Updated for update to our schema.

### PR DESCRIPTION
**Pull request by joerage**

### CODE REVIEW

File: schemas/2015-05-21-preview/Microsoft.DevTestLab.json [hunk 1]
```diff
@@ -21,13 +21,13 @@
       },
       "required": [ "type", "apiVersion", "location" ]
     },
-    "environments": {
-      "description": "Required properties to create environment in an existing DevTest Lab",
+    "virtualMachines": {
```
Comment (wenwu449) [inline schemas/2015-05-21-preview/Microsoft.DevTestLab.json]: When will your service start to support this resource type? This change will cause existing template be invalid.

Comment (joerage) [inline schemas/2015-05-21-preview/Microsoft.DevTestLab.json]: Service already support the virtualMachines resource type and environments has been removed. We can take breaking changes as we are still not GA. We announced that breaking change, and contacted the few customers who had not made changes.

File: schemas/2015-05-21-preview/Microsoft.DevTestLab.json [hunk 2]
```diff
@@ -21,13 +21,13 @@
...
+      "description": "Required properties to create a virtual machine in an existing DevTest Lab",
       "type": "object",
       "properties": {
         "type": {
           "enum": [
-            "Microsoft.DevTestLab/environments"
+            "Microsoft.DevTestLab/labs/virtualMachines"
```
Comment (wenwu449) [inline schemas/2015-05-21-preview/Microsoft.DevTestLab.json]: Please add/update tests.

File: schemas/2015-05-21-preview/Microsoft.DevTestLab.json [hunk 3]
```diff
@@ -38,38 +38,18 @@
...
-              "description": "Microsoft.DevTestLab/environments - VMs"
-            }
-          },
-          "required": [ "labId", "vms" ]
+            "customImageId": { "type": "string" },
+            "size": { "type": "string" },
+            "userName": { "type": "string" },
+            "password": { "type": "string" },
+            "sshKey": { "type": "string" },
+            "isAuthenticationWithSshKey": { "type": "boolean" },
```
Comment (wenwu449) [inline schemas/2015-05-21-preview/Microsoft.DevTestLab.json]: Please add support for expression.

File: schemas/2015-05-21-preview/Microsoft.DevTestLab.json [hunk 4]
```diff
@@ -38,38 +38,18 @@
         "properties": {
           "type": "object",
           "properties": {
```
Comment (wenwu449) [inline schemas/2015-05-21-preview/Microsoft.DevTestLab.json]: is there any of below properties are required?

File: schemas/2015-05-21-preview/Microsoft.DevTestLab.json (hunk 3 above)
Comment (joerage) [inline schemas/2015-05-21-preview/Microsoft.DevTestLab.json]: Done.

File: schemas/2015-05-21-preview/Microsoft.DevTestLab.json (hunk 4 above)
Comment (joerage) [inline schemas/2015-05-21-preview/Microsoft.DevTestLab.json]: Yes, added.

File: schemas/2015-05-21-preview/Microsoft.DevTestLab.json (hunk 2 above)
Comment (joerage) [inline schemas/2015-05-21-preview/Microsoft.DevTestLab.json]: Can you give me details about which tests are needed?

Comment (wenwu449) [inline schemas/2015-05-21-preview/Microsoft.DevTestLab.json]: At least a valid JSON body for each resource type, negative tests are welcome as well.
Please check /tests folder for other RPs' tests.

File: schemas/2015-05-21-preview/Microsoft.DevTestLab.json [hunk 5]
```diff
@@ -36,40 +36,26 @@
...
+          "oneOf": [		 
+            {
+              "type": "object",
+              "properties": {
+                "customImageId": { "type": "string" },
+                "size": { "type": "string" },
+                "userName": { "type": "string" },
+                "password": { "type": "string" },
+                "sshKey": { "type": "string" },
+                "isAuthenticationWithSshKey": { "type": "boolean" },
```
Comment (wenwu449) [inline schemas/2015-05-21-preview/Microsoft.DevTestLab.json]: this boolean property needs to support expression too.

File: schemas/2015-05-21-preview/Microsoft.DevTestLab.json [hunk 6]
```diff
@@ -38,38 +38,46 @@
         "properties": {
           "type": "object",
           "properties": {
-            "labId": {
+            "customImageId": { 
               "type": "string",
-              "minLength": 1,
-              "description": "Microsoft.DevTestLab/environments - Lab ID"
+              "description": "blah"
```
Comment (wenwu449) [inline schemas/2015-05-21-preview/Microsoft.DevTestLab.json]: Customer can see description when using Visual Studio, please use descriptive content.

Comment (joerage) [inline schemas/2015-05-21-preview/Microsoft.DevTestLab.json]: Fixed.

**Pull request merged**